rn/Teacher/src/modules/courses/settings/CourseSettings.js

```diff
@@ -137,7 +137,7 @@ export class CourseSettings extends Component<Props, State> {
                 </Text>
                 <TextInput
                   value={this.state.name}
-                  style={styles.actionableText}
```

Comment:
do we need to cleanup actionableText or it's still used?

Reply:
It's still used for a different row.